src/components/LocationTimeZoneSelector.tsx

```diff
@@ -0,0 +1,65 @@
+import { Checkbox } from '@terraware/web-components';
+import { useEffect, useState } from 'react';
+import strings from 'src/strings';
+import { TimeZoneDescription } from 'src/types/TimeZones';
+import { useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
+import TimeZoneSelector from './TimeZoneSelector';
+
+type TimeZoneEntity = {
+  timeZone?: string;
+};
+
+type LocationTimeZoneSelectorProps = {
+  onChangeTimeZone: (tzSelected: TimeZoneDescription | undefined) => void;
+  location: TimeZoneEntity;
+};
+
+export default function LocationTimeZoneSelector(props: LocationTimeZoneSelectorProps): JSX.Element {
+  const [orgTZChecked, setOrgTZChecked] = useState<boolean>(false);
```

Comment:
instead of false can we use `!!location.timeZone`

Reply:
This doesn't work, that's why I created the useEffect. I think location might be undefined at the beginning until the location object (the nursery in this case) is fetched